pkg/plugin/manager.go

```diff
@@ -20,7 +20,7 @@ import (
 // Note: the plugin directory is the /plugins inside the home directory.
 func Directory() string {
 	homeDir, _ := config.GetConfigDir()
-	pluginsDir := path.Join(homeDir, "my_plugins")
+	pluginsDir := path.Join(homeDir, "thyra-plugins")
```

Comment:
IMO, it should just be `plugins` since this directory is created inside `$HOME/.config/thyra/`

```suggestion
	pluginsDir := path.Join(homeDir, "plugins")
```

Reply:
@SlnPons wdyt?

Reply:
It looks logical as this folder will be under thyra. 
Can you confirm that it will be `thyra/plugins/...` please?

Reply:
yes it will be ~/.config/thyra/plugins/wallet-plugin

Reply:
Ok, good for me. Thanks guys :)